src/Model/Service.cs

```diff
@@ -19,14 +19,16 @@ public class Service
         /// <param name="enums">The enum types that are used by the service.</param>
         /// <param name="exceptions">The exception types that are used by the service.</param>
         /// <param name="xmlSequenceWrappers"></param>
-        public Service(string clientName, string clientDescription, IEnumerable<string> subpackages, IEnumerable<ServiceEnum> enums, IEnumerable<ServiceException> exceptions, IEnumerable<XmlSequenceWrapper> xmlSequenceWrappers, ServiceManager manager)
+        /// <param name="manager"></param>
+        public Service(string clientName, string clientDescription, IEnumerable<string> subpackages, IEnumerable<ServiceEnum> enums, IEnumerable<ServiceException> exceptions, IEnumerable<XmlSequenceWrapper> xmlSequenceWrappers, IEnumerable<ServiceModel> models, ServiceManager manager)
         {
             ClientName = clientName;
             ClientDescription = clientDescription;
             SubPackages = subpackages;
             Enums = enums;
             Exceptions = exceptions;
             XmlSequenceWrappers = xmlSequenceWrappers;
+            Models = models;
```

Comment:
Any reason why Settings is mutable while Service is immutable?

Reply:
It was easier at the start, but I think it makes sense to make it immutable. I'll make that change.

Reply:
Done.